packages/cli/src/commands/account/balance.ts

```diff
@@ -22,6 +22,7 @@ export default class Balance extends BaseCommand {
       goldBalance: await goldToken.balanceOf(args.address),
       dollarBalance: await stableToken.balanceOf(args.address),
     }
+    console.log('All balances expressed in units of 10^-18.')
```

Comment:
Not sure if this is "human readable" but it's better to what we have before.

Maybe check with @cla-bel

Reply:
Works for now.